src/server/implementation/types.ts

```diff
@@ -83,7 +83,14 @@ export const authTables = {
     sessionId: v.id("authSessions"),
     expirationTime: v.number(),
     firstUsedTime: v.optional(v.number()),
-  }).index("sessionId", ["sessionId"]),
+    parentRefreshTokenId: v.optional(v.id("authRefreshTokens")),
```

Comment:
Can you add some documentation above re: the parent relationship that's being tracked now?